api/pkg/service/catalog/catalog_test.go

```diff
@@ -120,3 +120,24 @@ func TestRefresh_All(t *testing.T) {
 	assert.Contains(t, []string{"catalog-official", "catalog-community"}, jobs[1].CatalogName)
 	assert.Equal(t, "queued", jobs[1].Status)
 }
+
+func TestCatalogError(t *testing.T) {
```

Comment:
Could you please add a test here when the errors occurs and assert the error message ?